chart/values.schema.json

```diff
@@ -2,68 +2,112 @@
     "$schema": "http://json-schema.org/draft-07/schema",
     "description": "Default values for airflow. Declare variables to be passed into your templates.",
     "type": "object",
+    "docs_section_order": [
```

Comment:
What do you think to use the `x-` prefix to denote that this field is not part of the official JSON Schema specification? This is not required in our case, but I think it will improve readability.

Reply:
I'm okay with that. Maybe `x-docsSectionOrder` and `x-docsSection`? Feels like we should use the `x-` prefix everywhere for non standard fields?

Reply:
Let's do it in the follow-up PR please

Reply:
#15828